app/helpers/defendant_link_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module DefendantLinkHelper
+  def defendant_link_path(defendant)
+    if defendant.linked?
```

Comment:
🥇

Reply:
You should unit test this though `spec/helpers/defendant_link_helper_spec.rb`

Reply:
I'd also rename this module to a more generic `DefendantHelper` and, to avoid name clashes and aid readability rename the DefendantHelper controller concern to something else

Reply:
👍 I have created spec/helpers/defendant_helper_spec.rb and renamed defendant_link_helper.rb defendant_helper.rb